packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -1,5 +1,5 @@
-import { Language } from '@logto/phrases';
 import { hexColorRegEx } from '@logto/shared';
+import { Language } from '@logto/ui-phrases/lib/types.js';
```

Comment:
https://stackoverflow.com/questions/30302747/root-directory-in-package-json
Nit: use this for shorter import

Reply:
updated